### ISSUE TITLE
Instabug shake only works on Android when phone has been locked + unlocked while app is running

### ISSUE THREAD

**Issue by saadq** [Instabug/Instabug-React-Native#233]
For whatever reason, shaking my device to open the Instabug dialog seems to only work when I lock + unlock my device while the app is running. If I close the app and reopen it, I would need to lock + unlock my device again while the app is running in order to get it to work. Otherwise, nothing happens.

### Steps to Reproduce the Problem

First, I ran the following:

```bash
npm install instabug-reactnative
react-native link instabug-reactnative
```

Then, I simply replaced the Instabug token in `MainApplication.java` and I added the 1 line in my `App.js` file to start Instabug.

### Expected Behavior
Instabug should not require lock + unlock in order to start.

### Actual Behavior
Instabug requires me to lock + unlock while the app is running in order for shaking to activate it. If I close the app, I need to redo the lock + unlock of my device.

### Instabug integration code
In my App.js, I have:

```js
import React from 'react'
import Instabug from 'instabug-reactnative'
import { INSTABUG_TOKEN } from 'react-native-dotenv'
...

function App() {
  Instabug.startWithToken(INSTABUG_TOKEN, [Instabug.invocationEvent.shake])
  return (
    <Provider store={store}>
      <Navigation />
    </Provider>
  )
}

export default App
```

In `MainApplication.java`, I have:

```java
// Needed for `react-native link`
public List <ReactPackage> getPackages() {
    return Arrays.<ReactPackage>asList(
        // Add your own packages here!
        // TODO: add native modules!

        // Needed for `react-native link`
        // new MainReactPackage(),
        new RNInstabugReactnativePackage.Builder("<my-instabug-token>", MainApplication.this)
        .setInvocationEvent("shake")
        .setPrimaryColor("#1D82DC")
        .setFloatingEdge("left")
        .setFloatingButtonOffsetFromTop(250)
        .build()
    );
}
```

### SDK Version
instabug-reactnative: v8.0.20

### React Native, iOS and Android Versions
React Native: v31.0.0
Android: Pie

### Device Model
It's happening on pretty much every device I'm trying it out on (both physical and Android Emulator).

### [Optional] Project That Reproduces the Issue
I can look into trying to create a minimal project if there isn't an obvious reason why this is happening.

**Comment by saadq** [comment]
Update – It doesn't actually have to be lock/unlock. Alternatively, minimizing the app and reopening it will also make Instabug run.

**Comment by salmamali** [comment]
@saadq We can't seem to reproduce this behavior. Would it be possible to send a sample app that has this issue?

**Comment by saadq** [comment]
Yeah, I'll try to see if i can create a small app that reproduces this.

**Comment by saadq** [comment]
Alright, I've created a project here: https://github.com/saadq/instabug-android-issue

This is just an app that I created using Expo and then ejected straight after. Lastly, I added the Instabug integration in `App.js` and `MainApplication.java`. You can see the 3 commits in the project to see each step in depth.

## How I created this project

1. I initialized the project by running `expo init instabug-android-bug`
2. I ejected from Expo by running `expo eject --eject-method expokit` which created an `ios` and `android` directory
3. I installed Instabug by running `npm install instabug-reactnative`
4. I ran `react-native link instabug-reactnative`
5. I added `Instabug.startWithToken(INSTABUG_TOKEN, [Instabug.invocationEvent.shake]);` to `App.js`
6. I updated the Instabug token in `android/app/src/main/java/host/exp/exponent/MainApplication.java`
7. Lastly, I made a single change to `android/build.gradle` by adding:

```gradle
subprojects {
  project.configurations.all {
    resolutionStrategy.eachDependency { details ->
      if (details.requested.group == 'com.android.support'
              && !details.requested.name.contains('multidex') ) {
        details.useVersion "26.1.0"
      }
    }
  }
}
```

I needed to do this because I was getting some dependency error.

## Running the app

From the root project folder, install dependencies by running:

```
npm install
```

Next, you'll need to start the expo server by running this in the root project folder:

```
npm start
```

Lastly, you'll want to open up the `android` directory in Android Studio and build/run the app there. Make sure that `npm start` has been run **before** trying to run the Android app from Android Studio.

Please let me know if I can provide anything else that can help!

**Comment by HeshamMegid** [comment]
Thanks for the project!

This is an issue with our Android SDK that has been fixed and will be available in the next release.
